lib/coverage-instrumenter.js

```diff
@@ -24,25 +25,35 @@ function getPathForRealFile(relativePath, root, templateExtensions) {
   return null;
 }
 
-function fixPath(relativePath, name, root, templateExtensions) {
+function fixPath(relativePath, name, root, templateExtensions, isAddon) {
   // Handle apps (served from app name)
-  if (relativePath.indexOf(name) === 0) {
-    relativePath = relativePath.replace(name, 'app')
-    return getPathForRealFile(relativePath, root, templateExtensions) || relativePath
+  if (!isAddon && relativePath.includes(name)) {
```

Comment:
Hmm, I think that `includes(name)` would return false positives wouldn't it? 

Consider running against an app named `ui`, addon files for an addon named `breadcrumb-ui` would be included...

I think we need to keep the `indexOf(name) === 0` or swap to `startsWith(name)`.